tests/Microsoft.Identity.Test.Unit/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -1808,6 +1808,52 @@ public async Task AcquireTokenForClientAuthorityCheckTestAsync()
             }
         }
 
+        [TestMethod]
+        public async Task AcquireTokenOboAuthorityCheckTestAsync()
```

Comment:
Should be refactored to use DataRow providing common and org authorities as params?